src/komponenter/header/Header.tsx

```diff
@@ -3,23 +3,32 @@ import Typer from "./typer/Typer";
 import SprakVelger from "./sprak/SprakVelger";
 import VeilederEttersendelse from "../veileder/varianter/Ettersendelse";
 import VeilederKlage from "../veileder/varianter/Klage";
+import { Sidetittel } from "nav-frontend-typografi";
+import { injectIntl, FormattedMessage, InjectedIntlProps } from "react-intl";
 
 interface Props {
-  visTyper?: boolean;
+  erForside: boolean;
   inngang?: string;
 }
 
-const Header = ({ visTyper, inngang }: Props) => (
+const Header = ({ erForside, inngang }: Props & InjectedIntlProps) => (
```

Comment:
Headeren brukes i to hovedcaser, det som er på forsiden, og "resten". Er ikke superfornøyd med propertynavnet her, men det er jo tydelig i det minste. Hva sier navnekonvensjonene @7er ?

Reply:
Flottington